src/main/java/frc/robot/commands/ScoringElevatorPositionCommand.java

```diff
@@ -9,6 +9,15 @@ public class ScoringElevatorPositionCommand extends Command {
     private ScoringElevator scoring = ScoringElevator.getInstance();
     private ElevatorSetpoints scoringState;
 
+    private boolean noFinish = false;
```

Comment:
I think the default of this should be true so that by default this command will not end until cancelled